src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmEvaluationAsyncResult.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#region Assembly Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
+// D:\Roslyn\Main\Open\Binaries\Debug\Microsoft.VisualStudio.Debugger.Engine.dll
+#endregion
+
+using System;
+
+namespace Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation
+{
+    public struct DkmEvaluationAsyncResult
+    {
+        public DkmEvaluationAsyncResult(DkmEvaluationResult Result)
+        {
+            throw new NotImplementedException();
+        }
+
+        public int ErrorCode { get { throw new NotImplementedException(); } }
+
+        public DkmEvaluationResult Result { get { throw new NotImplementedException(); } }
+
+        public static DkmEvaluationAsyncResult CreateErrorResult(Exception exception)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Will we call CreateErrorResult if we throw an exception from a DkmClrValue method such as EvaluateDebuggerDisplayString or GetMemberValue?